src/System.Private.Xml/src/System/Xml/Xslt/XslCompiledTransform.cs

```diff
@@ -214,47 +204,6 @@ private void CompileQilToMsil(XsltSettings settings)
         }
 
         //------------------------------------------------
-        // Compile stylesheet to a TypeBuilder
-        //------------------------------------------------
-
-        private static volatile ConstructorInfo s_generatedCodeCtor;
-
-        internal static CompilerErrorCollection CompileToType(XmlReader stylesheet, XsltSettings settings, XmlResolver stylesheetResolver, bool debug, TypeBuilder typeBuilder, string scriptAssemblyPath)
-        {
-            if (stylesheet == null)
-                throw new ArgumentNullException(nameof(stylesheet));
-
-            if (typeBuilder == null)
-                throw new ArgumentNullException(nameof(typeBuilder));
-
-            if (settings == null)
-                settings = XsltSettings.Default;
-
-            if (settings.EnableScript && scriptAssemblyPath == null)
-                throw new ArgumentNullException(nameof(scriptAssemblyPath));
-
-            if (scriptAssemblyPath != null)
-                scriptAssemblyPath = Path.GetFullPath(scriptAssemblyPath);
-
-            QilExpression qil;
-            CompilerErrorCollection errors = new Compiler(settings, debug, scriptAssemblyPath).Compile(stylesheet, stylesheetResolver, out qil).Errors;
-
-            if (!errors.HasErrors)
-            {
-                // Mark the type with GeneratedCodeAttribute to identify its origin
-                if (s_generatedCodeCtor == null)
-                    s_generatedCodeCtor = typeof(GeneratedCodeAttribute).GetConstructor(new Type[] { typeof(string), typeof(string) });
-
-                typeBuilder.SetCustomAttribute(new CustomAttributeBuilder(s_generatedCodeCtor,
-                    new object[] { typeof(XslCompiledTransform).FullName, Version }));
-
-                new XmlILGenerator().Generate(qil, typeBuilder);
-            }
-
-            return errors;
-        }
-
-        //------------------------------------------------
         // Load compiled stylesheet from a Type
         //------------------------------------------------
 
```

Comment:
@danmosemsft @weshaggard I just noticed that although System.CodeDom reference has been removed from this project, there is still one more usage of it in [XslCompiledTransform.Load(Type) [line 268] ](http://source.dot.net/#System.Private.Xml/System/Xml/Xslt/XslCompiledTransform.cs,267) which I'm currently missing in my changes here. 
The type being used is `System.CodeDom.GeneratedCodeAttribute`. Looking at the code I don't think we can just remove it. However, this method is supposed to load the compiled stylesheet that was created using the **XSLT Compiler (xsltc.exe)**. Since xsltc.exe is not ported to CoreFX [due to assembly generation missing from CodeDom], shouldn't we exclude `XslCompiledStylesheet.Load(Type) `from NS. 2.0 as well?